deploy/backplane/lpsre/config.yaml

```diff
@@ -1,36 +1,34 @@
 deploymentMode: "SelectorSyncSet"
 selectorSyncSet:
   matchLabels:
-      api.openshift.com/addon-rhmi-operator: "true"
-      api.openshift.com/addon-rhmi-operator-internal: "true"
-      api.openshift.com/addon-managed-api-service: "true"
-      api.openshift.com/addon-managed-api-service-internal: "true"
-      api.openshift.com/addon-kas-fleetshard-operator: "true"
-      api.openshift.com/addon-kas-fleetshard-operator-qe: "true"
-      api.openshift.com/addon-managed-kafka: "true"
-      api.openshift.com/addon-managed-kafka-qe: "true"
-      api.openshift.com/addon-managed-odh: "true"
-      api.openshift.com/addon-ocs-consumer: "true"
-      api.openshift.com/addon-ocs-converged: "true"
-      api.openshift.com/addon-ocs-provider: "true"
-      api.openshift.com/addon-reference-addon: "true"
-      api.openshift.com/addon-advanced-cluster-management: "true"
-      api.openshift.com/addon-cluster-logging-operator: "true"
-      api.openshift.com/addon-connectors-operator: "true"
-      api.openshift.com/addon-dbaas-operator: "true"
-      api.openshift.com/addon-klusterlet-operator: "true"
-      api.openshift.com/addon-managed-starburst: "true"
-      api.openshift.com/addon-mcg-osd: "true"
-      api.openshift.com/addon-mcg-osd-dev: "true"
-      api.openshift.com/addon-mcg-osd-qe: "true"
-      api.openshift.com/addon-nvidia-gpu-addon: "true"
-      api.openshift.com/addon-ocs-consumer-dev: "true"
-      api.openshift.com/addon-ocs-consumer-qe: "true"
-      api.openshift.com/addon-ocs-converged-dev: "true"
-      api.openshift.com/addon-ocs-converged-qe: "true"
-      api.openshift.com/addon-ocs-provider-dev: "true"
-      api.openshift.com/addon-ocs-provider-qe: "true"
-      api.openshift.com/addon-managed-api-service: "true"
-      api.openshift.com/addon-smart-events-operator: "true"
```

Comment:
this indentation change is hiding the *real* change here (which is the removal of `api.openshift.com/addon-managed-api-service: "true"` if I saw correctly)

Reply:
I've removed a duplicate, it's still there :)